docker-initunlocklnd.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+echo "[lnd_unlock] Waiting 2 seconds for lnd..."
+sleep 2;
+
+LND_DIR="/root/.lnd"
+CA_CERT="$LND_DIR/tls.cert"
+
+LND_WALLET_DIR="$LND_DIR/data/chain/$1/$2/"
+WALLET_FILE="$LND_WALLET_DIR/wallet.db"
+LNDUNLOCK_FILE=${WALLET_FILE/wallet.db/walletunlock.json}
+MACAROON_FILE="$LND_WALLET_DIR/admin.macaroon"
+MACAROON_HEADER="r0ckstar:dev"
```

Comment:
Not sure how macaroons work but does this have any repercussions? :P

Reply:
Nopoe, no repercussions... other than those line being better for having `r0ckstardev` in them.